src/Microsoft.OData.Client/DataServiceContext.cs

```diff
@@ -2559,7 +2559,7 @@ internal Task<QueryOperationResponse> LoadPropertyAllPagesAsync(object entity, s
         /// </summary>
         /// <param name="entity">The entity that contains the property to load.</param>
         /// <param name="propertyName">The name of the property of the specified entity to load.</param>
-        /// <returns>An instance of <see cref="T:Microsoft.OData.Client.QueryOperationResponse`1" /> that contains the results of the last page request.</returns>
+        /// <returns>An instance of <see cref="Microsoft.OData.Client.QueryOperationResponse`1" /> that contains the results of the last page request.</returns>

```

Comment:
I didn't resolve ALL warnings. I will split this across multiple pull requests, as otherwise it will get too big to review. There will still be warnings left after this pull request is completed. Either way, your comment is definitely valid, I added the change.